src/Response/Search.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Envato\Response;
+
+class Search {
+  protected $data;
+
+  public function __construct($response) {
+      // todo: Can we throw an exception here if json_decode fails (i.e. API returns invalid JSON like the the 500 unavaiable HTML error)
```

Comment:
Don't worry about it for now, I've got something in the works that is going to take care of this inside of the request prep so the endpoints themselves don't need to.